<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_einstellungen</class>
 <widget class="QDialog" name="Dialog_einstellungen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>351</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Anzahl der wiederrufen-Schritte in der Programmliste          </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_anz_ur_prg">
      <property name="minimum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_anz_ur_wkz">
      <property name="minimum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Anzahl der wiederrufen-Schritte in der Werkzeugliste</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>641</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit_userpath_opendialog"/>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="radioButton_stdpath_opendialog_post">
      <property name="text">
       <string>Zielverzeichnis des Postprozessors</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="radioButton_stdpath_opendialog_user">
      <property name="text">
       <string>Benutzer-Definierter Pfad:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Standart-Verzeichnis für den Datei-öffnen-Dialog:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_userpath_opendialog">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>281</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_entwicklermodus">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Entwicklermodus</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_anz_ur_prg</tabstop>
  <tabstop>spinBox_anz_ur_wkz</tabstop>
  <tabstop>radioButton_stdpath_opendialog_post</tabstop>
  <tabstop>radioButton_stdpath_opendialog_user</tabstop>
  <tabstop>lineEdit_userpath_opendialog</tabstop>
  <tabstop>pushButton_userpath_opendialog</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_abbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
